src/connectTool/makeTool.js

```diff
@@ -23,7 +23,7 @@ export const makeTool = (
 
   // checks if the locale has finished loading
   const isLocaleLoaded = () => {
-    return !hasLocale || getLocaleLoaded(store.getState());
```

Comment:
This is a little tricky.
Basically it's checking to see if the tool has any locale. If it does not then it will automatically finish loading the locale. Otherwise it will check if the locale as actually finished loading.

I'll look at the related issue.